<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="127096" started-at="2021-03-03T11:43:52 IST" finished-at="2021-03-03T11:45:59 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="127096" started-at="2021-03-03T11:43:52 IST" finished-at="2021-03-03T11:45:59 IST">
      <class name="testScripts.AssertStaging">
        <test-method status="FAIL" signature="createStaging()[pri:0, instance:testScripts.AssertStaging@68c9d179]" name="createStaging" duration-ms="125929" started-at="2021-03-03T11:43:52 IST" finished-at="2021-03-03T11:45:58 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [AM ICT User] but found [Asset Admin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [AM ICT User] but found [Asset Admin]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testScripts.AssertStaging.createStaging(AssertStaging.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createStaging -->
        <test-method status="PASS" signature="tearDownDriver()[pri:0, instance:testScripts.AssertStaging@68c9d179]" name="tearDownDriver" is-config="true" duration-ms="733" started-at="2021-03-03T11:45:59 IST" finished-at="2021-03-03T11:45:59 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownDriver -->
      </class> <!-- testScripts.AssertStaging -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
